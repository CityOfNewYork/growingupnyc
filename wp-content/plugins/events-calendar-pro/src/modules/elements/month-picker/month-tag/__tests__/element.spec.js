import React from 'react';
import renderer from 'react-test-renderer';
import MonthTag from '../element';

describe( 'MonthTag', () => {
	test( 'should render component', () => {
		const component = renderer.create(
			<MonthTag
				className="test-class"
				onClick={ jest.fn() }
			>
				Modern Tribe
			</MonthTag>
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
