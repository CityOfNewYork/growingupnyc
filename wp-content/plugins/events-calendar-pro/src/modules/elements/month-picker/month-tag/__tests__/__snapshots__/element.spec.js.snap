// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MonthTag should render component 1`] = `
<button
  className="tribe-editor__month-tag test-class"
  onClick={[MockFunction]}
>
  <<svg>icon mock</svg> />
  <span
    className="tribe-editor__month-tag__remove"
  >
    Remove
  </span>
  Modern Tribe
</button>
`;
